azure-storage-nspkg/setup.py

```diff
@@ -43,11 +43,11 @@
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
         'License :: OSI Approved :: MIT License',
     ],
     zip_safe=False,
     packages=[
-        'azure',
```

Comment:
was this not needed?

Reply:
because provided by azure-nspkg (this should have been like that in the first place)